src/libraries/System.ComponentModel.Composition/ref/System.ComponentModel.Composition.cs

```diff
@@ -507,15 +511,14 @@ public abstract partial class ComposablePart
         protected ComposablePart() { }
         public abstract System.Collections.Generic.IEnumerable<System.ComponentModel.Composition.Primitives.ExportDefinition> ExportDefinitions { get; }
         public abstract System.Collections.Generic.IEnumerable<System.ComponentModel.Composition.Primitives.ImportDefinition> ImportDefinitions { get; }
-        public virtual System.Collections.Generic.IDictionary<string, object> Metadata { get { throw null; } }
+        public virtual System.Collections.Generic.IDictionary<string, object?> Metadata { get { throw null; } }
         public virtual void Activate() { }
-        public abstract object GetExportedValue(System.ComponentModel.Composition.Primitives.ExportDefinition definition);
+        public abstract object? GetExportedValue(System.ComponentModel.Composition.Primitives.ExportDefinition definition);
         public abstract void SetImport(System.ComponentModel.Composition.Primitives.ImportDefinition definition, System.Collections.Generic.IEnumerable<System.ComponentModel.Composition.Primitives.Export> exports);
     }
     public abstract partial class ComposablePartCatalog : System.Collections.Generic.IEnumerable<System.ComponentModel.Composition.Primitives.ComposablePartDefinition>, System.Collections.IEnumerable, System.IDisposable
     {
         protected ComposablePartCatalog() { }
-        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
This has been removed by the tooling and should probably be reinstated.

Reply:
Sure, thanks